src/fromager/external_commands.py

```diff
@@ -9,9 +9,12 @@
 
 logger = logging.getLogger(__name__)
 
-NETWORK_ISOLATION: list[str] | None
+NETWORK_ISOLATION: list[list[str]] | None
 if sys.platform == "linux":
-    NETWORK_ISOLATION = ["unshare", "--net", "--map-current-user"]
+    NETWORK_ISOLATION = [
+        ["bwrap", "--unshare-network", "--dev-bind", "/", "/", "--"],
```

Comment:
--unshare-network doesn't appear to be valid?  Is this supposed to be --unshare-net ?

Reply:
FYI: this needs docker to specify --privileged in builder's bin/boostrap.sh